src/schemas/ev42/ev42_rw.h

```diff
@@ -0,0 +1,61 @@
+#include "../../FlatbufferReader.h"
+#include "../../HDFWriterModule.h"
+#include "../../h5.h"
+
+namespace FileWriter {
+namespace Schemas {
+namespace ev42 {
+template <typename T> using uptr = std::unique_ptr<T>;
+
+struct append_ret {
```

Comment:
This `struct` is only used in the implementation of these classes so it doesn't need to be in the header.